<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDeadBodyDialog</class>
 <widget class="QDialog" name="AddDeadBodyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 77, 56);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>280</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ageLineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="causeLineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>460</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateLineEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>370</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="heightLineEdit">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>460</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="assigngrave">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>520</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Segoe UI&quot;;
background-color: rgb(0, 77, 56);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Add body</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 13pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>520</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 77, 56);
font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>471</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0,0,0,0);
font: 22pt &quot;Segoe UI&quot;;
color: rgb(0, 77, 56);</string>
   </property>
   <property name="text">
    <string>Add Dead Body</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 13pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>340</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 13pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Date of Death:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 13pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Cause of death:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 13pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Height in cm:</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>nameLineEdit</zorder>
  <zorder>ageLineEdit</zorder>
  <zorder>causeLineEdit</zorder>
  <zorder>dateLineEdit</zorder>
  <zorder>heightLineEdit</zorder>
  <zorder>assigngrave</zorder>
  <zorder>name</zorder>
  <zorder>backButton</zorder>
  <zorder>label_2</zorder>
  <zorder>name_2</zorder>
  <zorder>name_3</zorder>
  <zorder>name_4</zorder>
  <zorder>name_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
